src/components/ResourceConditionField/index.jsx

```diff
@@ -25,6 +27,9 @@ function cleanText(value, operator) {
       newValue = newValue.slice(0, -1)
     }
   }
+  if(operator.toLowerCase() === 'in') {
+    newValue = newValue.join(',')
```

Comment:
I think that you need to add a check here to make sure newValue is a string with commas in it. As is with out a value this errors in the form when select OR as an option before entering text